CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [3.0.0] - 2023-11-09
 
+### Fixed
+- Fixed human model calibration bug in which sometimes it gets stuck (https://github.com/robotology/human-dynamics-estimation/pull/388).
```

Comment:
You should add the line on a different section, not in an already existing release. If you prefer you can also stop using the CHANGELOG.md, as we did for example in https://github.com/robotology/whole-body-estimators/pull/180 .